services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls(cinema_halls: int) -> CinemaHall:
```

Comment:
The return type hint for `get_cinema_halls` is incorrect. It should be `QuerySet[CinemaHall]` instead of `CinemaHall` because the function returns all cinema halls, not just one.